assets/src/web-stories-block/block/components/blockTypeSwitcher.js

```diff
@@ -33,18 +33,25 @@ import { BLOCK_TYPES } from '../constants';
 
 function BlockTypeSwitcher({ selectedBlockType, setAttributes }) {
   return (
-    <DropdownMenu
-      icon={update}
-      label={__('Change Block Type', 'web-stories')}
-      controls={BLOCK_TYPES.filter(
-        (blockType) => blockType.id !== selectedBlockType
-      ).map((blockType) => {
-        return {
-          title: blockType.label,
-          onClick: () => setAttributes({ blockType: blockType.id }),
-        };
-      })}
-    />
+    <Toolbar label={__('Change Block Type', 'web-stories')}>
+      <ToolbarItem>
+        {(toolbarItemHTMLProps) => (
+          <DropdownMenu
+            icon={update}
+            toggleProps={toolbarItemHTMLProps}
+            label={__('Change block type dropdown', 'web-stories')}
+            controls={BLOCK_TYPES.filter(
+              (blockType) => blockType.id !== selectedBlockType
+            ).map((blockType) => {
+              return {
+                title: blockType.label,
+                onClick: () => setAttributes({ blockType: blockType.id }),
+              };
+            })}
+          />
+        )}
+      </ToolbarItem>
+    </Toolbar>
```

Comment:
Looking at the [documentation](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/toolbar#development-guidelines), `<Toolbar label={__('Options', 'web-stories')` and `<DropdownMenu label={__('Block Type', 'web-stories')}` seems more accurate.

----


Aside:

Is there another word we could use instead of "block type"? Block types in Gutenberg are something different (`registerBlockType`, etc.), so this is a bit confusing.

Reply:
I'll update the labels.

For the `block type`, kept as is as these were supposed to be three separate block that we have merged together. Couldn't find more accurate name for this one. any suggestion?

Reply:
I have a hard time finding a good name as well.

Some ideas off the top of my head:

* contentType
* blockSubType
* blockContentType

Reply:
let's go with the `blockContentType`, this signifies what type of content the block has, is it picked by editors ( selected ) or latests ( general story river ).

Reply:
@swissspidy do we want to update the attributes and used variable names from `blockType` to `blockSubType` across the block implementation right.